ietf-optical-impairment-topology.tree

```diff
@@ -59,29 +67,32 @@ module: ietf-optical-impairment-topology
     |        +--ro OTSi-signal-width?        decimal64
     |        +--ro channel-delta-power?      decimal64
     +--ro transponders-list* [transponder-id]
-       +--ro transponder-id                      uint32
-       +--ro (mode)?
-       |  +--:(G.692.2)
-       |  |  +--ro standard-mode?                standard-mode
-       |  +--:(organizational-mode)
-       |  |  +--ro operational-mode?             operational-mode
-       |  |  +--ro organization-identifier?      vendor-identifier
-       |  +--:(explicit-mode)
-       |     +--ro available-modulation-types*   identityref
-       |     +--ro configured-modulation-type?   identityref
-       |     +--ro available-baud-rates*         uint32
-       |     +--ro configured-baud-rate?         uint32
-       |     +--ro available-FEC-types*          identityref
-       |     +--ro configured-FEC-type?          identityref
-       |     +--ro FEC-code-rate?                decimal64
-       |     +--ro FEC-threshold?                decimal64
-       +--ro power?                              int32
-       +--ro power-min?                          int32
-       +--ro power-max?                          int32
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+       +--ro transponder-id      uint32
+       +--ro tranceivers-list* [tranceiver-id]
+          +--ro tranceiver-id      uint32
+          +--ro supported-modes* [mode-id]
+          |  +--ro mode-id                          string
+          |  +--ro (mode)
+          |     +--:(G.692.2)
+          |     |  +--ro standard-mode?             standard-mode
+          |     +--:(organizational-mode)
+          |     |  +--ro operational-mode?          operational-mode
+          |     |  +--ro organization-identifier?   vendor-identifier
+          |     +--:(explicit-mode)
+          |        +--ro supported-std-modes
+          |           +--ro supported-application-codes*
+          |           |       -> ../../mode-id
+          |           +--ro supported-organizational-modes*
+          |                   -> ../../mode-id
+          +--ro power?             int32
+          +--ro power-min?         int32
+          +--ro power-max?         int32
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro transponder-list* [carrier-id]
        +--ro carrier-id    uint32
```

Comment:
what is the carrier-id in a flexgrid network? does it correspond to a channel (see OTSiG container)?

Reply:
Not related to the update in review: we have transponder-id in the update definiton. What you mention is realted to sliceable-transponder